py/server/setup.py

```diff
@@ -68,7 +68,7 @@ def _compute_version():
         'numba; python_version < "3.13"',
     ],
     extras_require={
-        "autocomplete": ["jedi==0.19.1"],
+        "autocomplete": ["jedi==0.19.1", "docstring_parser==0.16"],
```

Comment:
Like the base image, we should prefer to be more lenient. `>=0.16` is what I think you decided there.